### ISSUE TITLE
Adding support for Tezos blockchain

### ISSUE THREAD

**Issue by surajabdul99** [davehagler/nftlogin#4]
I have implemented support for Tezos blockchain, kindly let me know if it is something you would like to consider.

**Comment by davehagler** [comment]
Yes, that would be great. Please submit a pull request https://github.com/davehagler/nftlogin
